app/index.js

```diff
@@ -135,6 +118,23 @@ const notifyCertError = (webContents, url, error, cert) => {
 }
 
 app.on('ready', () => {
+  let loadAppStatePromise = SessionStore.loadAppState()
+
+  // Some settings must be set right away on startup, those settings should be handled here.
+  loadAppStatePromise.then((initialImmutableState) => {
+    const {HARDWARE_ACCELERATION_ENABLED, SMOOTH_SCROLL_ENABLED, SEND_CRASH_REPORTS} = require('../js/constants/settings')
+    CrashHerald.init(getSetting(SEND_CRASH_REPORTS, initialImmutableState.get('settings')))
+
+    telemetry.setCheckpointAndReport('state-loaded')
+    if (getSetting(HARDWARE_ACCELERATION_ENABLED, initialImmutableState.get('settings')) === false) {
+      app.disableHardwareAcceleration()
+    }
+
+    if (getSetting(SMOOTH_SCROLL_ENABLED, initialImmutableState.get('settings')) === false) {
+      app.commandLine.appendSwitch('disable-smooth-scrolling')
+    }
+  })
```

Comment:
I'm pretty sure that moving this on ready will make these not work

Reply:
https://github.com/brave/muon/pull/392/files

Reply:
also verified that smooth scrolling is correctly enabled/disabled